docs/bugs_and_glitches.md

```diff
@@ -1881,6 +1882,34 @@ This bug can affect Mew or Pokémon other than Ditto that used Transform via Mir
 ```
 
 
+## Having the Dude catch a Pokémon while your boxes and party are full shows the box is full message
+
+([Video](https://youtu.be/A8zaTOkjKS4?t=407))
```

Comment:
For consistency with other Youtube URLs: `https://www.youtube.com/watch?v=A8zaTOkjKS4&t=407`

Also, is it typical to see that graphical glitch too? Or is it specific to the German version? If it's different from English, we should have a video of English pokecrystal.

Reply:
According to https://glitchcity.info/wiki/Dude_full_box_glitch, yes

Reply:
@Rangi42 Both the German and the English versions (other versions too?) show the same graphical glitches, the former being able to use the resulting glitch as an infinite money trick for reasons that need to be investigated.

Reply:
Fixed the video link using @Rangi42's suggestion.